### PR TITLE
Create new blog post about Huis voor de Sport

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Add blog page
- [x] Add mood image
- [x] Add featured image
- [x] Add social image
- [x] Add screenshot(s)
- [x] Make new blog featured on homepage

### CODE REVIEW

File: source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Digitale omgeving met eigen look & feel
+Huis voor de Sport Groningen en CMO Stamm ontwikkelden voor toekomstige bestuurders, vrijwilligerscoördinatoren en -commissies een online cursusprogramma gericht op het trainen en begeleiden van sportverenigingen. Deelnemers volgen binnen hetzelfde platform een cursus, leveren opdrachten in en ontvangen hierop feedback van een e-coach. De verzameling van deze opdrachten vormt de basis van het beleidsplan voor hun vereniging. “Wij hebben hierbij voor CAPP Agile Learning gekozen omdat we geheel naar eigen inzicht een digitale omgeving kunnen inrichten, met onze eigen look & feel”, zegt Simone Smuling, Manager Sport en Bewegen van Huis voor de Sport Groningen.
```
Comment (fatboypunk) [inline source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: Do we want to link to CAPP Agile Learning?

Comment (michellusthof) [reply source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: Yeah, good suggestion.

Comment (RonaldDijkstra) [reply source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: Something we forgot in yesterday's blog too!

**Comment by RonaldDijkstra** [comment]
You shouldn't use target blank on internal links ;-)

File: source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## On demand leren en kennis delen
+Vrijwilligers zetten zich graag in voor hun vereniging, maar combineren dit vaak met een vaste baan. De resterende avonduren worden dan al snel ingevuld met verenigingswerk, waardoor er weinig tijd overblijft voor adequate scholing, laat staan in een klassikale setting. [CAPP Agile Learning](https://www.defacto.nl/capp-agile-learning/){:target="_blank"}{:title="CAPP Agile Learning"} stelt deelnemers in staat om on demand te leren: waar en wanneer zij dat willen. Geheel in hun eigen tempo. Daarnaast worden deelnemers gestimuleerd om via het tabblad “vragen” kennis te halen en te delen met elkaar.
+
+## Pilot
+Het huidige programma Vrijwilligers & Co bestaat uit een pilot waaraan gedurende 2019 dertig sportverenigingen kosteloos kunnen deelnemen.
+
+
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: Remove extra blank line

File: source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## Digitale omgeving met eigen look & feel
+Huis voor de Sport Groningen en CMO Stamm ontwikkelden voor toekomstige bestuurders, vrijwilligerscoördinatoren en -commissies een online cursusprogramma gericht op het trainen en begeleiden van sportverenigingen. Deelnemers volgen binnen hetzelfde platform een cursus, leveren opdrachten in en ontvangen hierop feedback van een e-coach. De verzameling van deze opdrachten vormt de basis van het beleidsplan voor hun vereniging. “Wij hebben hierbij voor [CAPP Agile Learning](https://www.defacto.nl/capp-agile-learning/){:target="_blank"}{:title="CAPP Agile Learning"} gekozen omdat we geheel naar eigen inzicht een digitale omgeving kunnen inrichten, met onze eigen look & feel”, zegt Simone Smuling, Manager Sport en Bewegen van Huis voor de Sport Groningen.
+
+![CAPP Agile Learning Huis voor de Sport Groningen](../images/blog/capp-agile-learning-huisvoordesport.jpg)
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: `CAPP Agile Learning Huis voor de Sport Groningen
`
should work just fine

File: source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Huis voor de Sport Groningen coacht Co met CAPP Agile Learning"
+date: 2019-03-22 08:00 CEST
+tags: CAPP Agile Learning
+author: Michel
+lang: nl
+featured: false
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: Do you want this or the one from yesterday at homepage?

Comment (michellusthof) [reply source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: We could probably replace the job opening with this one.

Comment (RonaldDijkstra) [reply source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: I'm actually talking about the featured blog section to the bottom, not the news section.

Comment (michellusthof) [reply source/blog/nl/2019-03-22-huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning.html.markdown]: I see. Let's keep that one for now.

**Pull request merged**